tests/shell/shell_recorder/tutorial_wrapper/README.md

```diff
@@ -2,7 +2,7 @@
 
 The Markdown Shell Recorder extracts executable shell snippets from Markdown files and translate them into a suitable format for the [`shell_recorder`](../shell_recorder/README.md).
 
-Snippets are shell commands inside a syntax block with additional checks (such as exit code, output, errors, etc) encoded as comments. These blocks start with ```` ```sh ````  and end with ```` ``` ````.
+Snippets are shell commands inside a syntax block with additional checks (such as exit code, output, errors, etc) encoded as comments. These blocks start with ` ```sh ` and end with ` ``` `.
```

Comment:
Does this work? Iirc the 4 ticks are needed.

Reply:
As far as I can tell, yes. At least [Pandoc](https://pandoc.org)

<img width="896" alt="pandoc" src="https://user-images.githubusercontent.com/691989/52914430-5efd0900-32c8-11e9-8926-32e97197dafc.png">

and [Marked](http://marked2app.com):

<img width="897" alt="marked" src="https://user-images.githubusercontent.com/691989/52914383-cd8d9700-32c7-11e9-8d71-1db1b1361743.png">

produce the correct output.